.github/workflows/pythonpublish.yml

```diff
@@ -0,0 +1,26 @@
+name: Upload Python Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
```

Comment:
It is just question - it can be like that: Do we really want always latest version? Isn't better to use specific LTS version